backend/src/main/java/com/eyetrade/backend/model/resource/user/MinimalUserResource.java

```diff
@@ -15,6 +16,8 @@
 @Resource
 public class MinimalUserResource {
 
+    private UUID id;
```

Comment:
I didn't understand why you are adding id fields to different types of user resources.

Reply:
Because if we didn't return id of user there is no chance to access to the user's profile at frontend side. They can send request only with id. I used minimal user resource for annotation resource. The edit is not only for my usage but also for other usages.